spec/services/support_interface/notifications_export_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::NotificationsExport do
+  describe '#data_for_export' do
+    let!(:provider1) { create(:provider, code: 'ABC') }
+    let!(:provider2) { create(:provider, code: 'XYZ') }
+    let!(:provider_user1) { create(:provider_user, providers: [provider1, provider2]) }
+    let!(:provider_user2) { create(:provider_user, providers: [provider1, provider2]) }
+
+    before do
+      ProviderUserNotificationPreferences.find_by_provider_user_id(provider_user1.id).update!(
+        application_received: false,
+        application_withdrawn: true,
+        application_rejected_by_default: false,
+        offer_accepted: true,
+        offer_declined: false,
+      )
+      ProviderUserNotificationPreferences.find_by_provider_user_id(provider_user2.id).update!(
+        application_received: true,
+        application_withdrawn: false,
+        application_rejected_by_default: true,
+        offer_accepted: false,
+        offer_declined: true,
+      )
+
+      provider_user1.provider_permissions.where(provider: provider1).update(make_decisions: true)
+    end
+
```

Comment:
can you also add `it_behaves_like 'a data export'`?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4898/commits/cd128cb3e0fbfc9adc64b6f0026825b2d3576ea6

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4898/files#diff-e436fed6a047e63636ad2aec65f2a619c89bff5fda53cf346aef46670501efb7R34